### DISCUSSION TITLE
Alternativas ao apidocjs para documentação de REST API

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#2342] [Perguntas e Respostas]
Oi pessoal, estou criando o primeiro tópico que menciona apidocjs no fórum, tendo tido uma experiência bem agradável com graphql pois o playground faz um ótimo trabalho em centralizar todos os recursos da api e seus respectivos parâmetros e retornos.

Porém, eu também mantenho uma REST API, e não tem uma documentação. Como a do graphql, é feita quase que automaticamente pela definição dos recursos no código, ao ver algo como o apidocjs me da uma sensação de que está faltando algo.

Com este contexto, eu vos pergunto: conhece alguma alternativa ao https://apidocjs.com que se integra mais profundamente ao express, para gerar documentações semi-automaticamente?

Obrigado desde já pela ajuda.

**Comment by gstcarv** [comment]
Já me recomendaram o `swagger-autogen`. Nunca cheguei a utilizar, mas pela documentação parece muito bom e aparentemente com uma configuração simples.

https://www.npmjs.com/package/swagger-autogen
